src/features/store.ts

```diff
@@ -33,11 +33,13 @@ export interface AppState {
   multiProvider: MultiProtocolProvider;
   registry: IRegistry;
   warpCore: WarpCore;
+  warpDeployConfig: WarpDeployConfigChainAddressMap;
```

Comment:
Let's not name things that are not `WarpDeployConfig` `warpDeployConfig`. Consider using `WarpDeployConfigIndex` or `WarpDeployConfigMap`